app/components/projects/index_page_header_component.html.erb

```diff
@@ -21,7 +21,7 @@
         )
       end
 
-      if query.persisted? && OpenProject::FeatureDecisions.project_list_sharing_active?
+      if query.persisted? && OpenProject::FeatureDecisions.project_list_sharing_active? && can_view_shares?
```

Comment:
I was thinking about this as well. But if the user cannot view the query, they will never reach the index page and thus this check does not really do anything.

Reply:
I just realized after committing too. Probably superfluous so I'll remove it